build.sbt

```diff
@@ -199,9 +199,9 @@ lazy val webapi: Project = Project(id = "webapi", base = file("webapi"))
     IntegrationTest / testForkedParallel := false, // not run forked tests in parallel
     IntegrationTest / parallelExecution  := false, // not run non-forked tests in parallel
     // Global / concurrentRestrictions += Tags.limit(Tags.Test, 1), // restrict the number of concurrently executing tests in all projects
-    IntegrationTest / javaOptions += "-Dconfig.resource=fuseki.conf",
-    // Test / javaOptions ++= Seq("-Dakka.log-config-on-start=on"), // prints out akka config
-    // Test / javaOptions ++= Seq("-Dconfig.trace=loads"), // prints out config locations
+    // IntegrationTest / javaOptions ++= Seq("-Dakka.log-config-on-start=on"), // prints out akka config
+    // IntegrationTest / javaOptions ++= Seq("-Dconfig.trace=loads"), // prints out config locations
+    // IntegrationTest / javaOptions += "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005", // starts sbt with debug port
```

Comment:
I find those commented-out lines a bit irritating. If they have to stay there, please add a comment why/how/what for they are needed, otherwise I'd suggest removing them

Reply:
In general I agree that commented out code should be removed. Those Java options are documented and seemed useful for me so I kept them.